rtloader/three/three.cpp

```diff
@@ -129,8 +132,10 @@ py_info_t *Three::getPyInfo()
         setError("could not allocate a py_info_t struct");
         return NULL;
     }
-
-    info->version = Py_GetVersion();
+    const char* v = Py_GetVersion();
+    if(v){
+        info->version = strdup(v);
```

Comment:
If we're going to use helper functions to get and free `py_info_t`, then there should be no confusion as to how to properly free the structure as the wrapper will now handle the specifics. I feel like we don't need this allocation to be honest. `Py_GetVersion` will point to static storage that should be available all throughout the lifecycle of the python interpreter which in our case is pretty much the same as the lifetime of the agent itself so it should be totally safe.

This would of course apply to `Two` below as well.

Reply:
Yeah, I reverted that back. Was going for consistency, but the free function removes that a bit.